rosetta-source/src/main/rosetta/observable-asset-enum.rosetta

```diff
@@ -325,3 +325,7 @@ enum CsaTypeEnum: <"How is the Creadit Support Annex defined for this transactio
 enum OptionReferenceTypeEnum: <"The enumeration values to specify the reference source that determines the final settlement price of the option.">
     Future <"Reference from the price of a future contract.">
     Spot <"Reference from an underlyer spot price.">
+	
+enum PriceScheduleReferenceEnum: <"Describes the type and method that defines how the principal payment is set.">
```

Comment:
Given the fundament role of price the definition for each of these enum values if accepted, should be expanded. At the moment the description is circular and it is difficult to understand usage.